DOCUMENTATION.md

```diff
@@ -0,0 +1,121 @@
+Smart cards | `--socket=pcsc` and `--nosocket=pcsc` | Toggle | Permit (`--socket=pcsc`) or prohibit (`--nosocket=pcsc`) access to the application to smart cards.
+Printing system | `--socket=cups` and `--nosocket=cups` | Toggle | Permit (`--socket=cups`) or prohibit (`--nosocket=cups`) access to the application to printing systems.
+
+#### Device
+
+List of devices available in the sandbox.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+GPU acceleration | `--device=dri` and `--nodevice=dri` | Toggle | Permit (`--device=dri`) or prohibit (`--nodevice=dri`) access to the application to graphics direct rendering located at `/dev/dri`.
+Virtualization | `--device=kvm` and `--nodevice=kvm` | Toggle | Permit (`--device=kvm`) or prohibit (`--nodevice=kvm`) access to the application to virtualization.
+Shared memory | `--device=shm` and `--nodevice=shm` | Toggle | Permit (`--device=shm`) or prohibit (`--nodevice=shm`) access to the application to shared memory.
+All devices | `--device=all` and `--nodevice=all` | Toggle | Permit (`--device=all`) or prohibit (`--nodevice=all`) access to the application to all devices.
+
+#### Allow
+
+List of features available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+Development syscalls | `--allow=devel` and `--disallow=devel` | Toggle | [NEEDS DESCRIPTION]
+Programs from other architectures | `--allow=multiarch` and `--disallow=multiarch` | Toggle | [NEEDS DESCRIPTION]
+Bluetooth | `--allow=bluetooth` and `--disallow=bluetooth` | Toggle | Permit (`--allow=bluetooth`) or prohibit (`--disallow=bluetooth`) access to the application to use bluetooth.
+Controller Area Network bus | `--allow=canbus` and `--disallow=canbus` | Toggle | [NEEDS DESCRIPTION]
+
+#### Filesystem
+
+List of filesystem subsets available to the application.
+
+Name | `flatpak override` equivalent | Type | Description
+--- | --- | --- | ---
+All filesystem files | `--filesystem=host` and `--nofilesystem=host` | Toggle | Permit (`--filesystem=host`) or prohibit (`--nofilesystem=host`) read-write access to the application to the whole filesystem. If it is permitted, the application has read-write access to every file and folder owned by you, the user. The rest will be read-only.
+All system libraries, executables and static data | `--filesystem=host-os` and `--nofilesystem=host-os` | Toggle | [NEEDS DESCRIPTION]
+All system configurations | `--filesystem=host-etc` and `--nofilesystem=host-etc` | Toggle | [NEEDS DESCRIPTION]
+All user files | `--filesystem=home` and `--nofilesystem=home` | Toggle | Permit (`filesystem=home`) or prohibit (`--nofilesystem=home`) read-write access to the application to the user directory (`$HOME`).
+Other files | `--filesystem=[DIRECTORY]`, `--filesystem=[DIRECTORY]:ro` and `--nofilesystem=[DIRECTORY]` | Input | [NEEDS DESCRIPTION]
```

Comment:
For the `:ro`, `:rw` and `:create` modifiers, you can find descriptions here https://docs.flatpak.org/page/sandbox-permissions-reference.html